daemon/list.go

```diff
@@ -499,14 +499,14 @@ func includeContainerInList(container *container.Container, ctx *listContext) it
 	networkExist := fmt.Errorf("container part of network")
 	if ctx.filters.Include("network") {
 		err := ctx.filters.WalkValues("network", func(value string) error {
-			if _, ok := container.NetworkSettings.Networks[value]; ok {
-				return networkExist
-			}
```

Comment:
@thaJeztah  I have removed exact match because if two network one with net1 and net111 present,
and each having one container. I we filtered it with net1  and we added exact match scenario then it'll just return container with net1 however net1 is also comes in net111 so ideally it should have list down both container when we filter with net1.

Do we want to add exact match then we can add this ?